rust/types/src/metadata.rs

```diff
@@ -360,15 +371,35 @@ pub enum WhereClauseComparator {
 }
 
 #[derive(Clone, Debug, PartialEq)]
-pub enum WhereClauseListOperator {
+pub enum SetOperator {
     In,
     NotIn,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum MetadataSetValue {
+    Bool(Vec<bool>),
+    Int(Vec<i64>),
+    Float(Vec<f64>),
+    Str(Vec<String>),
+}
+
+impl MetadataSetValue {
```

Comment:
Please use the Into/From TryInto/TryFrom traits for these sorts of conversions

https://doc.rust-lang.org/rust-by-example/conversion/from_into.html

Reply:
I removed this impl because it is used only once in a future PR, so I simply in-lined it.